cmd/manager/main.go

```diff
@@ -139,31 +140,28 @@ func main() {
 	}
 
 	// Setup Scheme for all resources
-	if err := mapiv1beta1.AddToScheme(mgr.GetScheme()); err != nil {
+	if err := machinev1beta1.Install(mgr.GetScheme()); err != nil {
```

Comment:
why not `machinev1.Install`?

Reply:
I have renamed the import from `mapiv1beta1 "github.com/openshift/api/machine/v1beta1"` to `machinev1beta1 "github.com/openshift/api/machine/v1beta1"` so added `machinev1.Install` at line no: 147

Reply:
My thinking was v1 contains Power VS Provider conf and v1beta1 contains machine and machineset crds. so i added both but i think we need to install v1beta1 for registering machine and machineset crds and avoid installing v1, Need to check

Reply:
removed machinev1.Install in the latest commit